### PR TITLE
Advisor - Refactored the code snippet

### PR DESCRIPTION
**Pull request by jawache**
Add code example for Advisor

#sign-off as per @TomArcher

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/advisor.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-1/docs-ref-services/advisor.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C11%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707111112195252-52%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/advisor.md [hunk 1]
```diff
@@ -23,16 +23,30 @@ With Advisor, you can:
...
+const msRestAzure = require('ms-rest-azure');
+const advisorManagement = require('azure-arm-advisor');
+
+msRestAzure.interactiveLogin().then( credentials => {
+  const client = new advisorManagement(credentials, 'your-subscription-id');
+  client.recommendations.list().then(recommendations => {
+    console.log('List of recommendations:');
+    console.dir(recommendations, {depth: null, colors: true});
+  });
+}).catch(err => {
```
Comment (johnpapa) [inline docs-ref-services/advisor.md]: For most demos I applaud error handling. But for the APIs I wonder if it is better and more readable to err on the side of brevity/readability and remove the error handling.

Thoughts?

Comment (jawache) [inline docs-ref-services/advisor.md]: Yeah good point, should be in the deeper reference material but not the overview snippet.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/advisor.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-1/docs-ref-services/advisor.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C11%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707111235252340-52%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@jawache Your changes seem to be inconsistent with the changes that @johnpapa is making. For example, you reversed the headings to show Management as an H2 and Install as an H3. This is reverse of what we're doing with the other articles. If I'm mistaken, please advise. #hold-off

**Comment by jawache** [comment]
My understanding is that each resource should have a management (and a client section if it makes sense to have one). 

Since the npm install will be different between management and client sections (the example code and samples will also be different) then it makes sense to have something like this:

- Client
  - npm Install
  - Example
  - Samples

- Management
  - npm Install
  - Example
  - Samples

vs.

-  npm Install
  - Management
  - Client

-  Example
  - Management
  - Client

-  Samples
  - Management
  - Client

@johnpapa thoughts?

**Comment by jawache** [comment]
@thedanfernandez any thoughts on this?

**Comment by TomArcher** [comment]
This is inconsistent with the other 45 Node service pages, as well as the other DevHubs (Java, Python, and .NET). 

**Please let's not change the standard pattern agreed upon without first having a discussion as it makes for an inconsistent experience for the customer.**

To be clear, I'm fine with either approach, but we should use a single patter for all service pages.

**Comment by jawache** [comment]
Noted, we've got a standup today will raise and get a consensus for how to proceed.

**Comment by johnpapa** [comment]
Let's be consistent for now @jawache  .... and I'll add this to a small but growing list of things we may want to do in a future pass

**Comment by TomArcher** [comment]
Changing back to #sign-off per the conversation with Carol, et al.

**Pull request merged**